migrations.ts

```diff
@@ -179,7 +180,7 @@ process.on('unhandledRejection', reason => {
   throw reason
 })
 
-if (!module.parent) {
+if (require.main === module) {
```

Comment:
What's going on here?

Reply:
`module.parent` is deprecated. `module.parent` would be defined if another file imported/required this file, otherwise if it was run directly (`node migrations.ts`) it would be undefined. But now you have to compare the `main` to the current module and if they are the same then that means the file was run directly